---
title: 「Cloud Manager Assets のパフォーマンステストが失敗しました」
description: Cloud Manager Assets のパフォーマンステストが失敗する理由を説明します。
solution: Experience Manager
product: Experience Manager
applies-to: "Experience Manager"
keywords: "KCS, cloud manager，アセットのパフォーマンス，エラー，Experience Manager"
resolution: Resolution
internal-notes: "https://jira.corp.adobe.com/browse/CQDOC-21090"
bug: false
article-created-by: Tanay Sharma .
article-created-date: "12/5/2023 3:46:03 PM"
article-published-by: Tanay Sharma .
article-published-date: "12/5/2023 4:22:43 PM"
version-number: 2
article-number: KA-23312
dynamics-url: "https://adobe-ent.crm.dynamics.com/main.aspx?forceUCI=1&pagetype=entityrecord&etn=knowledgearticle&id=ed605461-8593-ee11-be37-6045bd006b25"
source-git-commit: 9c8ef7a6e8fd758a8ced2beb397a780b7b70869c
workflow-type: tm+mt
source-wordcount: '280'
ht-degree: 2%

---

# Cloud Manager Assets のパフォーマンステストが失敗する


Assets のパフォーマンステストが、不明なエラーで失敗する。 その理由を学びます。

## 説明 {#description}


### 環境

AEM Managed Services

### 問題/症状 

Assets のパフォーマンステストが、不明なエラーで失敗する。

<b>ステップ失敗</b>: assetsTest

失敗したステップの assetsTest からの直接ログは、次のエラーメッセージに示すように使用できません。

`{"type":"http://ns.adobe.com/adobecloud/missing-step-logs","status":404,"title":"Step logs not found","errors":[ "Logs not found for programId=809, executionId=3473820, phaseId=6253498, stepId=11687695"] } `

ただし、pipelineExecutionFailureReasons は `ASSETS_TESTS_AUTHOR_CONNECTION_ERROR` エラーコードを含む `002_010_006` そして次のメッセージが表示されます。

`Unable to connect to the author instance. `

内部の詳細は、HTTP 応答の問題と、 *見つかりません*：通常、テストの実行中に、オーサーインスタンスのエンドポイントに到達できなかったことを示します。

主な問題は、 assetsTest 手順中のオーサーインスタンスへの接続エラーで、テストが正常に完了しなかったようです。

ただし、Cloud Manager ユーザーは存在し、作成者に手動でアセットをアップロードすると、期待どおりに動作します。

アセットのパフォーマンステストが 30 分間動作した後、30 分後にテストが停止する間に、突然、繰り返し失敗します。

詳しくは、 [AEM Assets](https://experienceleague.adobe.com/docs/experience-manager-cloud-manager/content/using/code-quality-testing.html#aem-assets)(AEM 6.x の Cloud Manager ドキュメント )


## 解決策 {#resolution}


内部 Azure 関数ログを調べていると、リクエストの結果が「404 Not Found」であることが見つかりました。 GETから URL に *メッセージ* フィールドは 404 を返した。 以下を参照してください。

この場合の失敗は、Adobeの推奨に従って、querybuilder が Dispatcher レベルでブロックされたことが原因でした。
ただし、Cloud Manager では、Cloud Manager ユーザーの Dispatcher で querybuilder を許可する必要があります。アセットのテスト中に処理時間を取得するのと同じようにです。




```
{
 "x-ms-workflow-run-id": " ",
 "x-ms-workflow-operation-name": "1500_measureAssetsProcessingTime",
 "message": "Req: GET https://author-URL.adobecqms.net/bin/querybuilder.json?p.hitwriter=selective&p.limit=1000&p.properties=jcr%3Acontent%2Fjcr%3AlastModified+jcr%3Acreated&path=%2Fcontent%2Fdam%2Fcloudmanager&property=jcr%3Acontent%2Fdam%3AassetState&property.depth=1&property.value=processed&type=dam%3AAsset",
 "x-ms-client-tracking-id": "trackingID",
 "trackingContext": {
 "executionId": executionID,
 "tenantId": tenantID,
 "imsOrgId": "IMS_ORG",
 "programId": programID,
 "pipelineId": pipelineID
 }
}
```

